### PR TITLE
3rd localisation crusade

### PR DESCRIPTION
**Pull request by arithon**
## Changes for English:
- minor changes in wc_base_innovations_l_english.yml
- one scope change in wc_modifiers_l_english.yml
- one change in wc_character_interactions_l_english.yml (LK's Voice)
- small fix in wc_draenor_l_english.yml
- one scope changed in wc_game_concepts_l_english.yml (twisting nether return)
- some fixes for wc_holy_orders_l_english.yml
- small fixes in wc_regiment_l_english.yml

## Changes for French:
- bookmark_508_l_french added
- wc_base_innovations_l_french added
- wc_culture_eras_l_french added
- wc_regional_innovations_l_french added
- wc_being_events_l_french added
- wc_modifiers_l_french added
- wc_general_opinions_l_french added
- wc_opinions_l_french added
- wc_custom_titles_l_french added
- wc_racial_game_concepts_l_french added
- wc_regions_l_french added
- wc_succession_laws_l_french added
- wc_cultural_innovations_l_french updated
- wc_culture_titles_l_french updated
- wc_insult_custom_loc_l_french updated
- wc_dynasty_names_l_french updated
- wc_draenor_events_l_french updated
- wc_gui_l_french updated
- wc_holy_orders_l_french updated
- wc_portraits_l_french updated
- wc_traits_l_french updated
- wc_wars_l_french updated

## Changes for German:
- wc_cultural_innovations_l_german.yml
- wc_culture_eras_l_german.yml added
- wc_insult_custom_loc_l_german.yml added/done
- wc_death_reasons_l_german.yml updated
- ~3900 dynasty names added in wc_dynasty_names_l_german.yml
- one quotation fixed in wc_modifiers_l_german.yml
- wc_gods_and_titles_l_german.yml updated
- small fix in wc_effects_l_german.yml
- wc_game_concepts_l_german.yml updated
- wc_game_rules_l_german.yml updated
- fixed adjectives for dark governments in wc_government_l_german.yml
- wc_holy_orders_l_german.yml added
- quotations and line breaks in wc_traits_l_german.yml fixed
- quotation in wc_replacement_l_german.yml fixed
- wc_base_innovations_l_german added
- wc_magic_doctrines_l_german updated

### CODE REVIEW

File: localization/english/wc_holy_orders_l_english.yml [hunk 1]
```diff
@@ -406,7 +406,7 @@
  holy_order_firstborn_elementalists_article:1 "the "
  holy_order_firstborn_elementalists:1 "Firstborn Elementalists"
  holy_order_earth_wind_and_fire_article:0 "the "
- holy_order_earth_wind_and_fire:0 "Earth, Wind and Fire"
+ holy_order_earth_wind_and_fire:0 "Storm, Earth and Fire"
```
Comment (TheGroxEmpire) [inline localization/english/wc_holy_orders_l_english.yml:409]:
```suggestion
 holy_order_earth_wind_and_fire:0 "Storm, Earth, and Fire"
```

File: localization/english/wc_game_concepts_l_english.yml [hunk 2]
```diff
@@ -64,12 +64,12 @@
...
- game_concept_knowledge_overflow_common_desc:0 "When you unlock new [perks|E] the character may feel $game_concept_knowledge_overflow$: [lifestyle_experience|E] slow downs per each [perk|E].\n\nThis pace depends on [life_expectancy|E] of [wc_race|E]. Long-lived [wc_races|E] slow down slower, but sooner. Short-lived slow down faster, but later."
- game_concept_knowledge_overflow_desc:0 "$game_concept_knowledge_overflow_common_desc$\n\nFor example, [GetTrait('creature_human').GetName( GetNullCharacter )] characters slow down by #N [GuiScope.SetRoot(GetPlayer.MakeScope).ScriptValue('long_lived_lifestyle_xp_slowdown_human_per_perk_percent_value')|2]%#! per each [perk|E] starting after [GuiScope.SetRoot(GetPlayer.MakeScope).ScriptValue('long_lived_lifestyle_xp_slowdown_human_offset_value')|V0] [perk|E]."
- 
- 
+ game_concept_knowledge_overflow_common_desc:0 "When you unlock new [perks|E] the character may feel $game_concept_knowledge_overflow$: [lifestyle_experience|E] slows down per each [perk|E].\n\nThis pace depends on the [life_expectancy|E] of a [wc_race|E]. Long-lived [wc_races|E] slow down slower, but at a younger age. Short-lived slow down faster, but at a later age."
```
Comment (TheGroxEmpire) [inline localization/english/wc_game_concepts_l_english.yml:69]:
```suggestion
 game_concept_knowledge_overflow_common_desc:0 "When you unlock new [perks|E] the character may feel $game_concept_knowledge_overflow$: [lifestyle_experience|E] gain slows down per unlocked [perk|E].\n\nThis pace depends on the [life_expectancy|E] of a [wc_race|E]. Long-lived [wc_races|E] slow down slower, but at a younger age. Short-lived [wc_races|E] slow down faster, but at a later age."
```

File: localization/english/wc_draenor_l_english.yml [hunk 3]
```diff
@@ -13,5 +13,5 @@
  
  activity_beyond_dark_portal:0 "Invasion of Draenor"
  activity_beyond_dark_portal_name:1 "[ACTIVITY.GetOwner.GetShortUINamePossessive|U] Invasion of Draenor"
- activity_beyond_dark_portal_desc:0 "Every brave men and women has been invited to end with the Dark Portal once and for all!"
+ activity_beyond_dark_portal_desc:0 "Every brave man and woman has been invited to end the Dark Portal once and for all!"
```
Comment (TheGroxEmpire) [inline localization/english/wc_draenor_l_english.yml:16]:
```suggestion
 activity_beyond_dark_portal_desc:0 "Every brave man and woman has been invited to close the Dark Portal once and for all!"
```

File: localization/english/modifiers/wc_modifiers_l_english.yml [hunk 4]
```diff
@@ -41,21 +41,21 @@
...
  closed_from_another_side:0 "Closed from Another Side"
  closed_from_another_side_desc:0 "The Dark Portal is closed from another side."
- 
+
  game_start_health:0 "Long-lived"
- 
+
  long_lived_lifestyle_xp_slowdown:0 "[knowledge_overflow|E]"
- long_lived_lifestyle_xp_slowdown_custom_desc:0 "$game_concept_knowledge_overflow_common_desc$\n\nDue to [ROOT.Char.GetShortUINamePossessive] [life_expectancy|E], [lifestyle_experience|E] slow downs by #N [GuiScope.SetRoot(ROOT.Char.MakeScope).ScriptValue('long_lived_lifestyle_xp_slowdown_per_perk_percent_value')|2]%#! per each [perk|E] starting after [GuiScope.SetRoot(ROOT.Char.MakeScope).ScriptValue('long_lived_lifestyle_xp_slowdown_offset_value')|V0] [perk|E].\n\nPer stack:"
+ long_lived_lifestyle_xp_slowdown_custom_desc:1 "$game_concept_knowledge_overflow_common_desc$\n\nDue to [ROOT.Char.GetShortUINamePossessive] [life_expectancy|E], [lifestyle_experience|E] slow downs by #N [Scope.ScriptValue('long_lived_lifestyle_xp_slowdown_per_perk_percent_value')|2]%#! per each [perk|E] starting after [Scope.ScriptValue('long_lived_lifestyle_xp_slowdown_offset_value')|V0] [perk|E].\n\nPer stack:"
```
Comment (TheGroxEmpire) [inline localization/english/modifiers/wc_modifiers_l_english.yml:58]:
```suggestion
 long_lived_lifestyle_xp_slowdown_custom_desc:1 "$game_concept_knowledge_overflow_common_desc$\n\nDue to [ROOT.Char.GetShortUINamePossessive] [life_expectancy|E], [lifestyle_experience|E] gain slow downs by #N [Scope.ScriptValue('long_lived_lifestyle_xp_slowdown_per_perk_percent_value')|2]%#! per each unlocked [perk|E] starting after [Scope.ScriptValue('long_lived_lifestyle_xp_slowdown_offset_value')|V0] [perk|E].\n\nPer stack:"
```

**Review by TheGroxEmpire — COMMENTED** [review]
Few things that may clarify the tooltip.

File: localization/english/wc_holy_orders_l_english.yml (hunk 1 above)
Comment (arithon) [reply localization/english/wc_holy_orders_l_english.yml:409]: Shit, yeah, I somehow have to get that into my brain, to add the , before and ^^'

File: localization/english/wc_game_concepts_l_english.yml (hunk 2 above)
Comment (arithon) [reply localization/english/wc_game_concepts_l_english.yml]: fixed

File: localization/english/wc_draenor_l_english.yml (hunk 3 above)
Comment (arithon) [reply localization/english/wc_draenor_l_english.yml:16]: done

File: localization/english/modifiers/wc_modifiers_l_english.yml (hunk 4 above)
Comment (arithon) [reply localization/english/modifiers/wc_modifiers_l_english.yml:58]: fixed

**Review by TheGroxEmpire — APPROVED** [review]
The rest are good to me.

**Review by IronyOwl — APPROVED** [review]
The long lived lifestyle perk should say "gain slows down by" rather than "gain slows downs by", but otherwise looks good on the English side.

**Review by LilPenpusher — CHANGES_REQUESTED** [review]
One part needs to be changed (seems you skipped it by accident) + the mana addict part we discussed on Discord. Gotta see if that works in-game or not, haven't had time to check yet myself.

File: localization/german/wc_game_concepts_l_german.yml [hunk 5]
```diff
@@ -65,13 +65,21 @@
...
+ game_concept_knowledge_overflow_desc:0 "$game_concept_knowledge_overflow_common_desc$\n\nZum Beispiel verlangsamt sich der Erfahrungsgewinn von [GetTrait('creature_human').GetName( GetNullCharacter )] Charakteren um #N [GuiScope.SetRoot(GetPlayer.MakeScope).ScriptValue('long_lived_lifestyle_xp_slowdown_human_per_perk_percent_value')|2]%#! für jeden [Perk|E], beginnend nach [GuiScope.SetRoot(GetPlayer.MakeScope).ScriptValue('long_lived_lifestyle_xp_slowdown_human_offset_value')|V0] [perks|E]."
+
+
+ game_concept_wc_races:0 "Races"
+ game_concept_eldritch_government_desc:0 "A [dark_government|E] available to [GetTrait('being_void').GetName( GetNullCharacter )] with $doctrine_shadow_magic_name$ accepted.\nHaving high [dark_authority|E], they get all [skills|E] per [stress_level|E] and increase [county_development_growth|E]."
```
[the remark this answers is not in the document]
Comment (arithon) [reply localization/german/wc_game_concepts_l_german.yml:74]: yep, missed that (or was merged into masters after i did this file initially). fixed now.

File: localization/german/custom_localization/wc_insult_custom_loc_l_german.yml [hunk 6]
```diff
@@ -1,11 +1,46 @@
...
+ being_demon_adjective_article_female:0 "eine elende"
+ 
+ being_void_adjective_article_male:0 "ein gesichtsloser"
+ being_void_adjective_article_female:0 "eine gesichtslose"
+ 
+ being_life_adjective_article_male:0 "ein baumschmusender"
+ being_life_adjective_article_female:0 "eine baumschmusende"
+ 
+ being_order_adjective_article_male:0 "ein manasüchtiger"
+ being_order_adjective_article_female:0 "eine manasüchtiger"
```
[the remark this answers is not in the document]
Comment (arithon) [reply localization/german/custom_localization/wc_insult_custom_loc_l_german.yml:43]: aaand this one was most likely auto-fill from using notepad++ and i missed it ^^' fixed as well.

**Review by LilPenpusher — APPROVED** [review]

File: localization/french/custom_localization/wc_insult_custom_loc_l_french.yml [hunk 7]
```diff
@@ -1,9 +1,46 @@
 ﻿l_french:
  #InsultNoun
  rotting_corpse:0 "corps putride"
+ being_demon_insult_noun:0 "rejeton du Néant"
+ being_void_insult_noun:0 "abomination"
+ being_life_insult_noun:0 "forniqueur de cerfs"
+ being_order_insult_noun:0 "junkie"
```
Comment (runtd) [inline localization/french/custom_localization/wc_insult_custom_loc_l_french.yml:7]: Not a fan of keeping "junkie" in French as it does not fit with the fantasy universe of wow, you might want to change it for something like "addict" or "accro"

Comment (hamjustine) [reply localization/french/custom_localization/wc_insult_custom_loc_l_french.yml]: i added "addict" as you said and fix "forniqueur" as "fornicateur", also, i added customLoc, so those nouns are dynamics (Fornicatrice if female, fornicateur if male, etc etc)

**Review by runtd — APPROVED** [review]
Everything looks fine on the French side.

**Pull request merged**